joomla/joomla-cms
# [com_finder] index view: translate content types (solves #10260 - part 1)

#### Summary of Changes

Solves https://github.com/joomla/joomla-cms/issues/10260 (content types part)

> Load another language than English in back-end.
> Go to Smartsearch Indexed Content equivalent page
> The equivalent filter "- Select type of content -" displays the filter types in English

After PR (example in portuguese):

![image](https://cloud.githubusercontent.com/assets/9630530/15074977/fd631590-1399-11e6-9767-14d46848dfad.png)
#### Testing Instructions
1. Apply patch
2. Change admin language to other language than en-GB
3. Check the "Content Type" filter in Components -> Smart Search -> Indexed Content
